integration/basic_test.go

```diff
@@ -223,6 +223,26 @@ func TestDeleteCommentQuery(t *testing.T) {
 	assert.Equal(t, expected, res)
 }
 
+func TestEmptyKey(t *testing.T) {
```

Comment:
Added integration test to make sure that we don't skip empty keys by accident because of current pushdowns or some future ones